enterprise/developerbox/installation/README.md

```diff
@@ -48,7 +48,7 @@ Choose host machine to create the USB stick from:
 
 ## Boot into the installer and install OS
 
-**Note: On the Developer Box main board, flip switch SW3-2 to "Off" prior to starting installation.**
+**Note: On the Developer Box main board, power off the DevBox and flip switch SW3-2 to "Off" prior to starting installation.**
```

Comment:
I had forgotten about this bit of the docs. It definitely needs to be updated.

However...

Developerbox is the spelling used throughout the manual. It is more unique from a web search point of view; Developer Box and Devbox are too generic.

Additionally these workaround instructions are too specific to go here. They certainly won't work for Debian Stretch, CentOS or RHEL and perhaps not for Fedora either. I think the best we can in this generic section of to follow up with something like:

Note: *Some operating systems may require workarounds to fix known installation problems. See the [install guides for third party operating systems](https://www.96boards.org/documentation/enterprise/developerbox/downloads/#third-party-operating-systems) or, if your OS not listed, the [Known Issues](https://www.96boards.org/documentation/enterprise/developerbox/support/known-issues.html) for more details.*

Obviously the above would then have to be followed up with appropriate changes to the install guides for each OS.